comparisons/matrix_RE.txt

```diff
@@ -371,7 +383,13 @@ TTbar14TeVPUppmx2021wf11834p99 11834.99_TTbar_14TeV+*2021PU*/step4.root RECO
 TTbar14TeVPUppmx2021nanoEDMwf11834p99 11834.99_TTbar_14TeV+*2021PU*/step6.root NANO
 TTbarFS14TeV2021wf13234p0 13234.0_TTbar_14TeV+*2021*FS*/step1.root RECO
 TTbarFS14TeVPU2021wf13434p0 13434.0_TTbar_14TeV+*2021*FS*PU*/step1.root RECO
-TTbar14TeVPU2022nanoEDMin126Xwf2500p601 2500.601_mc126X+TTBarMINIAOD*/step2.root NANO
+TTbar14TeVmc122XnanoV10wf2500p400 2500.4_mc122Xrun3_v10+TTbarMINIAOD*/step2.root NANO
+TTbar14TeVmc122Xnanowf2500p401 2500.401_mc122Xrun3+TTbarMINIAOD*/step2.root NANO
+TTbar14TeVmc124XnanoV10wf2500p500 2500.5_mc124Xrun3_v10+TTbarMINIAOD*/step2.root NANO
+TTbar14TeVmc124Xnanowf2500p501 2500.501_mc124Xrun3+TTbarMINIAOD*/step2.root NANO
+TTbar14TeVmc126XnanoV10wf2500p600 2500.6_mc126X_v10+TTBarMINIAOD*/step2.root NANO
+TTbar14TeVmc126Xnanowf2500p601 2500.601_mc126X+TTBarMINIAOD*/step2.root NANO
```

Comment:
```suggestion
TTbar13p6TeVPU2022in122XnanoV10wf2500p4 2500.4_mc122Xrun3_v10+TTbarMINIAOD*/step2.root NANO
TTbar13p6TeVPU2022in122Xnanowf2500p401 2500.401_mc122Xrun3+TTbarMINIAOD*/step2.root NANO
MinBias14TeV2022in124XnanoV10wf2500p5 2500.5_mc124Xrun3_v10+TTbarMINIAOD*/step2.root NANO
MinBias14TeV2022in124Xnanowf2500p501 2500.501_mc124Xrun3+TTbarMINIAOD*/step2.root NANO
TTbar14TeVPU2022in126XnanoV10wf2500p6 2500.6_mc126X_v10+TTBarMINIAOD*/step2.root NANO
TTbar14TeVPU2022in126Xnanowf2500p601 2500.601_mc126X+TTBarMINIAOD*/step2.root NANO
```
please keep the year or at least the era in the names.
Note that .5 and .501 use `/MinBias_TuneCP5_14TeV-pythia8/Run3Summer22MiniAODv3-NoPU_Pilot` as input. I'm not sure which was the intention. If the fix to the workflow is imminent, then leaving TTbar may be better, otherwise MinBias would be more clear when inspecting the differences

Reply:
it is intended to be a TTBar sample, indeed